dotnet-desktop-guide/net/winforms/printing/snippets/how-to-print-windows-form/csharp/Form1.cs

```diff
@@ -0,0 +1,30 @@
+namespace Sample_print_win_form1
+{
+    public partial class Form1 : Form
+    {
+        Bitmap memoryImage;
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        private void Button1_Click(object sender, EventArgs e)
+        {
+            //<capture_screen>
```

Comment:
If the whole file is being referenced, the //<identifier> needs to be removed. Same for the VB file.